Pacman/Ghost.pde

```diff
@@ -0,0 +1,51 @@
+final String itemTypeGhost = "Ghost";
+String ghostImagePath = "data/ghost.jpg";
+PImage ghostImage;
+import java.util.Random;
+
+// Ghost class
+public class Ghost extends MovableItem {
+    private Random random = new Random();
+    private int ghostLife;
+    
+    // Constructor
+    public Ghost(String name, float x, float y) {
```

Comment:
the new constructor has parameters: `name`, `w`, `h`, where w is width, and h is height.
while position `x` and `y` need to be manually set.
so it’s better for the parameter names to be w and h, instead of x and y.